WonderBewegung.py

```diff
@@ -0,0 +1,261 @@
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                elif "ost" in benutzereingabe:
+                     self.bewegeEbene(benutzerWort)
+                    
+                else:
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                    print("Wenn Du nicht nach Westen weiter willst, musst du erst wieder einen Schritt zurück nach Osten laufen!")                    
+            else: 
+                 self.bewegeEbene(benutzerWort)
+
+            return False  
+
+    def bewege(self, eingabe, bewegungsWort:Wort):
+        self.eingabe =  eingabe
+        
+        self.uebergang = self.pruefeUebergang()
+        self.wonderText.setzeText("")
+
+        if  self.uebergang == False:    
+            wort = self.umgebung.vergleicheWorte(eingabe)
+            if wort is None:
+                self.wonderText.ergaenzeText("Achtung,in diesem Gebiet (" + str(self.umgebung.gebeBezeichnung())+ ") kann man nicht " + bewegungsWort.gebeBezeichnung() + ".")
+            elif "kleineres" not in self.grenzTest:
+                    self.wonderText.ergaenzeText ( self.umgebung.gebeUebergangssatz(self.grenzTest, self.uebergangstyp))                 
+                    self.uebergang = True                                 
+            elif "kleineres" in self.grenzTest:
+                # Bewegung durchführen
+                if wort.gebeWortTyp() == WortTyp.Flaeche:
+                    self.bewegeFläche(wort)             
+                   
+                elif wort.gebeWortTyp() == WortTyp.Ebene:
+                    self.bewegeEbene(wort)          
+                   
+                ##erneut prüfen, um wenn man jetzt an der Grenze ist Übergangssatz anzuzeigen
+                self.uebergang = self.pruefeUebergang()
+                if "kleineres" not in self.grenzTest:
+                    self.wonderText.ergaenzeText ( self.umgebung.gebeUebergangssatz(self.grenzTest, self.uebergangstyp))                  
+                    self.uebergang = True
+            else:
+                ##Übergangsatz anzeigen
+                self.wonderText.setzeText(self.umgebung.gebeUebergangssatz(self.grenzTest, self.uebergangstyp))
+                self.uebergang = True
+
+             ##Position am Ende angeben
+            self.wonderText.ergaenzeText(self.position.gebePositionsAusgabe())
+
+        elif self.uebergang == True:         
+            if self.praktiziereUebergang(eingabe,bewegungsWort,self.grenzTest, self.uebergangstyp) == True:
+                ##Übergang durchführen             
+                self.umgebung = self.umgebung.gebeNaechsteUmgebung(self, self.uebergangstyp)
+            
+            self.wonderText.ergaenzeText (self.position.gebePositionsAusgabe())
+            self.uebergang = False
+            
+
+    #Himmelsrichtungen
+
+    def nord(self, geschwindigkeit):
```

Comment:
Bei Methoden bietet sich immer an, ein Verb in den Namen mit aufzunehmen. Dann ist es verständlicher was sie eigentlich tun. Ich würde hieraus `bewegeNachNorden` machen oä.